testing/chain.go

```diff
@@ -137,14 +138,15 @@ func NewTestChain(t *testing.T, chainID string) *TestChain {
 	header := tmproto.Header{
 		ChainID: chainID,
 		Height:  1,
-		Time:    globalStartTime,
+		Time:    coord.CurrentTime.UTC(),
```

Comment:
NOTE: using non-UTC time can cause issues with proto (like if you use `time.Now()` which uses your local clock)